### ISSUE TITLE
Adding accessibility

### ISSUE THREAD

**Issue by margusholland** [WenchaoD/FSPagerView#244]
Any plans on adding accessibility to the PagerView? Cannot add it as an extension or subclass either, as the only public property is currentIndex and cannot determine when accessibilityIncrement and accessibilityDecrement should reset index to maximum or 0.

Also, it seems that calling scrollToItem do not call delegate methods e.g. pagerViewWillEndDragging not be called.
